Dockerfile

```diff
@@ -1,3 +1,4 @@
+# syntax=docker/dockerfile:1.3
```

Comment:
Should we use the `1.4`? Till now we were the latest which is last pushed `3 months ago`. 
`1.3` got pushed last `12 months back`?

Reply:
good point